<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileManagerWidget</class>
 <widget class="QWidget" name="ProfileManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="configProfileList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="profileAdd">
       <property name="toolTip">
        <string>Creates a new profile</string>
       </property>
       <property name="text">
        <string>New profile</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonFollowStyle</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="profileClone">
       <property name="toolTip">
        <string>Duplicate selected profile</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="edit-copy">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnExport">
       <property name="toolTip">
        <string>Export profile to file</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="document-save-as"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="profileDelete">
       <property name="toolTip">
        <string>Delete selected profile</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionImport">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Import from file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
